Dragonflight/WarriorArms.lua

```diff
@@ -572,6 +610,17 @@ spec:RegisterAuras( {
         copy = "pile_on"
     }
 })
+
+spec:RegisterSetBonuses( "tier29_2pc", 393705, "tier29_4pc", 393706 )
+--(2) Set Bonus: Mortal Strike and Cleave damage and chance to critically strike increased by 10%.
+--(4) Set Bonus: Mortal Strike, Cleave, & Execute critical strikes increase your damage and critical strike chance by 5% for 6 seconds.
+    spec:RegisterAuras( {
```

Comment:
If you are only registering one aura, you can use:

```Lua
spec:RegisterAura( "strike_vulnerabilities", {
    id = 394173,
    duration = 6,
    max_stack = 1
} )
```

`RegisterAuras` just unpacks the table and feeds things to `RegisterAura`.  Not really an issue, just for awareness.

Reply:
swapped to registeraura in https://github.com/Hekili/hekili/pull/1670/commits/3fd0a2231b6428c0aeb2056bd7cf895d82965dba